### DISCUSSION TITLE
Github checks

### DISCUSSION THREAD

**Discussion by luis-n4489** [concourse/concourse#8625] [General]
Is there a resource out there to add github check output for concourse? I saw this one but no clue how its used https://github.com/RoboJackets/concourse-github-check-resource

**Comment by marco-m** [comment]
For the [GitHub Checks](https://docs.github.com/en/rest/checks) API, I don't know, the resource you mention seems to be fit for the usage.
For the [GitHub Commit Status API](https://docs.github.com/en/rest/commits/statuses), the [Cogito](https://github.com/Pix4D/cogito) resource works well (I am one of the authors).
